snap/snapcraft.yaml

```diff
@@ -336,10 +335,15 @@ apps:
     plugs: [network, network-bind]
   app-service-configurable:
     adapter: full
-    command: bin/app-service-configurable -confdir $SNAP_DATA/config/app-service-configurable/res -profile rules-engine --registry
+    command: >-
+      bin/app-service-configurable
+      -confdir $SNAP_DATA/config/app-service-configurable/res
+      -profile rules-engine
+      -cp http://localhost:8500
```

Comment:
if you use this form the value needs to be `consul.http://localhost:8500`, which is the default if you just use `-cp` with no value. I recommend just `-cp` with no value. keep it simple... ;-)

Reply:
OK, it seemed to work fine w/out the `consul.`, but I'll change per your suggestion.

Why is the `consul.` needed? Is that a valid URI?

Reply:
that is the spec we agreed on so the provider type can be specified in the URL. BootStrap parses it out.

Reply:
Right, but the [config seed removal spec](https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/design/adr/0005-Service-Self-Config.md) interchangeably uses URI and URL, and never actually defines the supported schemes. What I was reacting to was the dotted scheme (eg. consul.http), as wasn't sure a scheme could include a "." character. Turns out it can per [RFC 3986 section 3.1](https://tools.ietf.org/html/rfc3986#section-3.1), so looks like we're OK. That said, it would be nice to enumerate the supported schemes someplace.